### ISSUE TITLE
Test suite requires `wget` to be present

### ISSUE THREAD

**Issue by awilfox** [scop/bash-completion#334]
The [Adélie Linux](https://www.adelielinux.org/) distribution does not ship `wget`.

This causes the bash-completion 2.9 test suite to fail:

```
_____________________________________________________ TestWget.test_2 _____________________________________________________

self = <test_wget.TestWget object at 0x3fff8b325470>, completion = <CompletionResult []>

    @pytest.mark.complete("wget --h")
    def test_2(self, completion):
>       assert completion
E       assert <CompletionResult []>

../t/test_wget.py:11: AssertionError
```

**Comment by awilfox** [comment]
It turns out I had a shim script in `/usr/bin` on my local machine to redirect wget usage to curl.  This isn't an issue with bash-completion.  Terribly sorry.

**Comment by scop** [comment]
No problem, good to hear tests being run on more distros. And this prompted some test suite fixes anyway, including 503143bf4dd7c43088b646f81eafa4494c476b26
